frontend/src/contexts/MeetingContext.tsx

```diff
@@ -0,0 +1,46 @@
+import { createContext, useContext, useEffect, useState } from "react";
+import MeetingDTO from "../shared/Types/dtos/MeetingDTO";
+import Permission from "../shared/Types/utils/Permission";
+import { useUserContext } from "./UserContext";
+
+export interface MeetingContextProps {
+    meetings: Record<string, MeetingDTO>,
+}
+
+export const useMeetingContext = () => useContext(MeetingContext);
+
+const MeetingContext = createContext<MeetingContextProps>({
+    meetings: {},
+});
+
+export function MeetingContextProvider({ children }) {
+    const userContext = useUserContext();
```

Comment:
I just realised we should be typing this, so it should be: `({ children }: { children?: ReactNode })`.